xla/service/gpu/transforms/stream_attribute_annotator.h

```diff
@@ -47,8 +47,10 @@ namespace xla::gpu {
 class StreamAttributeAnnotator : public HloModulePass {
  public:
   explicit StreamAttributeAnnotator(
-      const se::DeviceDescription& device_description)
-      : device_description_(device_description) {}
+      const se::DeviceDescription& device_description,
+      bool annotate_copy_start = false)
```

Comment:
Please do not use a default value and initialize this explicitly everywhere. Also please explain in a code comment what exactly this parameter does and when we would like to set it to false and when to true.

I'm not entirely familiar with this pass, but a general question is: is there a way to infer this value from the HLO during the pass, as opposed to explicitly controlling the behavior via a constructor parameter?